setup.cfg

```diff
@@ -5,12 +5,11 @@ universal = 1
 [pycodestyle]
 count = True
 # E124: Closing bracket does not match indentation of opening bracket's line
-# E128: Continuation line under-indented for visual indent
 # E261: At least two spaces before inline comment
 # E502: The backslash is redundant between brackets
 # W503: Line break occurred before a binary operator
 # W504: Line break occurred after a binary operator
-ignore = E124,E128,E261,E502,W503,W504
+ignore = E124,E261,E502,W503,W504
```

Comment:
I would allow failing on at least E124, E261, E502

Reply:
These were just to try and get my local checks failing in the same way as you were seeing with flake8 but without making too many unrelated changes. Let's take that up in https://github.com/microsoft/MLOS/pull/358